src/Compatibility/Microsoft.DotNet.PackageValidation/build/Microsoft.DotNet.PackageValidation.targets

```diff
@@ -3,27 +3,22 @@
   <UsingTask TaskName="Microsoft.DotNet.PackageValidation.GetLastStablePackage" AssemblyFile="$(DotNetPackageValidationAssembly)" />
 
-  <PropertyGroup>
-    <!-- On static graph build, the CollectPackageDownloads target is invoked per target and there is no other way to run logic only once for all tfms.
-         Hence the GetLastStablePackage feature should not be used when using static graph build as it results in a remote call per tfm. -->
-    <AddPackageValidationBaselinePackageAsPackageDownloadBeforeTargets Condition="'$(RestoreUseStaticGraphEvaluation)' == 'true'">CollectPackageDownloads</AddPackageValidationBaselinePackageAsPackageDownloadBeforeTargets>
-    <!-- For a non static graph restore with a single tfm, hook onto CollectPackageDownloads which will then be invoked just once. -->
-    <AddPackageValidationBaselinePackageAsPackageDownloadBeforeTargets Condition="'$(AddPackageValidationBaselinePackageAsPackageDownloadBeforeTargets)' == '' and '$(TargetFrameworks)' == '' and '$(TargetFramework)' != ''">CollectPackageDownloads</AddPackageValidationBaselinePackageAsPackageDownloadBeforeTargets>
-    <!-- For a non static graph restore with multiple tfms, hook onto the target that fans out to the individual tfm restore graph walk operations. -->
-    <AddPackageValidationBaselinePackageAsPackageDownloadBeforeTargets Condition="'$(AddPackageValidationBaselinePackageAsPackageDownloadBeforeTargets)' == ''">_GenerateProjectRestoreGraphAllFrameworks</AddPackageValidationBaselinePackageAsPackageDownloadBeforeTargets>
-  </PropertyGroup>
+  <ItemGroup Condition="'$(EnablePackageBaselineValidation)' == 'true' and '$(PackageValidationBaselinePath)' == '' and '$(PackageValidationBaselineVersion)' != ''">
+    <PackageDownload Include="$(PackageId)" Version="[$(PackageValidationBaselineVersion)]" />
+  </ItemGroup>
 
   <Target Name="RunPackageValidation"
           AfterTargets="Pack"
           Condition="$(IsPackable) == 'true'">
-    <PropertyGroup Condition="'$(EnablePackageBaselineValidation)' == 'true' and
-                              '$(PackageValidationBaselinePath)' == ''">
-      <PackageValidationBaselineName Condition="'$(PackageValidationBaselineName)' == ''">$(_PackageValidationBaselineName)</PackageValidationBaselineName>
-      <PackageValidationBaselineVersion Condition="'$(PackageValidationBaselineVersion)' == ''">$(_PackageValidationBaselineVersion)</PackageValidationBaselineVersion>
-      <PackageValidationBaselinePath Condition="'$(PackageValidationBaselineVersion)' != ''">$([MSBuild]::NormalizePath('$(NuGetPackageRoot)', '$(PackageValidationBaselineName.ToLower())', '$(PackageValidationBaselineVersion)', '$(PackageValidationBaselineName.ToLower()).$(PackageValidationBaselineVersion).nupkg'))</PackageValidationBaselinePath>
-      <EnablePackageBaselineValidation Condition="'$(PackageValidationBaselinePath)' == ''">false</EnablePackageBaselineValidation>
```

Comment:
Per @ViktorHofer's feedback in email we can set `EnablePackageBaselineValidation` to true when it is not set and either `PackageValidationBaselineVersion` or `PackageValidationBaselinePath` is set.

Basically minimize the number of things that the user needs to set in order to opt-into the comparison.